internal/providers/terraform/azure/testdata/notification_hubs_test/notification_hubs_test.golden

```diff
@@ -0,0 +1,33 @@
+
+ Name                                                   Monthly Qty  Unit                  Monthly Cost 
+                                                                                                        
+ azurerm_notification_hub_namespace.basicabove10M                                                       
+ ├─ Namespace usage (Basic)                                       1  months                      $10.00 
+ └─ Additional pushes (Over 10M)                                  2  1M pushes                    $2.00 
```

Comment:
This does not look correct.  The monthly_pushes in the usage file:
```
  azurerm_notification_hub_namespace.basicabove10M:    monthly_pushes: 20000000 # Monthly total number number of additional pushes.
```
the monthly_pushes is 20M so this line should be:
```
└─ Additional pushes (Over 10M)                                  10  1M pushes                    $10.00 
```